test/mitmproxy/addons/test_script.py

```diff
@@ -84,6 +85,25 @@ def test_reload(self, tmpdir):
                 time.sleep(0.1)
             tctx.master.has_log("Loading")
 
+    def test_deletion(self):
+        with open(tutils.test_data.path("mitmproxy/data/addonscripts/") + "/dummy.py", 'w') as f:
+            f.write("\n")
+        with taddons.context() as tctx:
+            sc = script.Script(
+                tutils.test_data.path("mitmproxy/data/addonscripts/dummy.py")
+            )
+            tctx.master.addons.add(sc)
+            tctx.configure(sc)
+
+            sl = script.ScriptLoader()
```

Comment:
Ok, but why do you need to instantiate `Script`, can't that be done by `ScriptLoader`?